### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [boazronkin/WareHouse#1]
It looks like you're missing most of your alt attributes in this file. Just as a reminder alt attributes are beneficial when it comes to increasing the accessibility of your page as they are used by screen readers to describe various images on your website. 

https://github.com/boazronkin/WareHouse/blob/562d2c64962a1084845db198888f72cdb542bbb0/tables.html#L63
